documentation/src/docs/asciidoc/user-guide/running-tests.adoc

```diff
@@ -755,3 +755,28 @@ useful.
 | +(micro \| integration) & (foo \| baz)+
 | all _micro_ or _integration_ tests for *foo* or *baz*
 |===
+
+[[running-tests-capturing-output]]
+=== Capturing Standard Output/Error
+
+Since version 1.3, the JUnit Platform provides opt-in support for capturing output
+printed to `System.out` and `System.err`. To enable it, simply set the
+`junit.platform.launcher.capture.stdout` and/or `junit.platform.launcher.capture.stderr`
```

Comment:
I think the prefix should be `junit.platform.launcher.output.capture` or `junit.platform.output.capture`.

Reply:
Will do!